src/library_pthread.js

```diff
@@ -992,49 +967,12 @@ var LibraryPThread = {
   },
 
   pthread_getschedparam: function(thread, policy, schedparam) {
```

Comment:
these can move to library_pthread.c to avoid the JS bounce.

Reply:
Resolved with https://github.com/kleisauke/emscripten/commit/c59a38f10bc7d380ee4d0eb214ca12f04fc3bbc8.